### PR TITLE
Portfolio covers

### PR DESCRIPTION
**Pull request by FelixLetheren**

### CODE REVIEW

File: stylesheets/stylesheet.css [hunk 1]
```diff
@@ -397,11 +416,20 @@ a {
         width: 350px;
     }
 
+    .portfolio-cover {
+        width: 350px;
+        height: 262.5px;
```
Comment (sjr-mayden) [inline stylesheets/stylesheet.css]: Just use whole numbers, they're already really small

File: index.html [hunk 2]
```diff
@@ -28,40 +28,65 @@ <h1>Portfolio</h1>
...
             <a href="https://github.com/FelixLetheren/mayden-academy-logo"><img class="portfolio-img"
                                                                                 src="images/mayden-logo.png"
-                                                                                alt="Mayden Academy Logo"></a>
-            <a href="https://github.com/FelixLetheren/mayden-academy-logo"><h3>Mayden Academy logo</h3></a>
+                                                                                alt="Mayden Academy Logo">
+                <div class="portfolio-cover">
+                    <h2 class="portfolio-cover-link">GitHub link</h2>
+                </div>
+            </a>
+            <a href="https://github.com/FelixLetheren/mayden-academy-logo"><h3>Mayden Academy
```
Comment (sjr-mayden) [inline index.html]: Open these links in a new tab for consistent behaviour

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

File: stylesheets/stylesheet.css (hunk 1 above)
Comment (FelixLetheren) [reply stylesheets/stylesheet.css]: Done!

File: index.html (hunk 2 above)
Comment (FelixLetheren) [reply index.html]: Boom, done

**Pull request merged**